src/main/java/edu/ucsb/cs156/gauchoride/entities/RideRequest.java

```diff
@@ -0,0 +1,31 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "riderequests")
+public class RideRequest {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long userid;
+  private String day;
+  private String fullName;
```

Comment:
This should not be here.  Get the name though the id.  Also, it's ambiguous: is it the rider name or the driver name?  Keep this in mind as you work; we'll eventually be tracking both.